packages/clerk-js/src/ui/elements/Action/ActionOpen.tsx

```diff
@@ -1,16 +1,29 @@
 import type { PropsWithChildren } from 'react';
+import React, { useEffect, useRef } from 'react';
 
 import { useActionContext } from './ActionRoot';
 
 type ActionOpenProps = PropsWithChildren<{ value: string }>;
 
+const ScrollWrapper = React.forwardRef<HTMLDivElement, PropsWithChildren>((props, ref) => {
+  return <div ref={ref}>{props.children}</div>;
```

Comment:
We can do `<div ref={ref} {...props} />` here.

Reply:
Done!